### PR TITLE
Update inventoried and scanned resources

### PR DESCRIPTION
**Pull request by mirons-google**
Thanks for opening a Pull Request!

Here's a handy checklist to ensure your PR goes smoothly.

- [x] I signed Google's [Contributor License Agreement](https://opensource.google.com/docs/cla/)
- [x] My code conforms to Google's [python style](https://google.github.io/styleguide/pyguide.html).
- [x] My PR at a minimum doesn't decrease unit-test coverage (if applicable).
- [x] All of the [unit-tests](http://forsetisecurity.org/docs/development/#executing-tests) still pass
- [x] Running `pylint --rcfile=pylintrc` passes.

These guidelines and more can be found in our [contributing guidelines](https://github.com/GoogleCloudPlatform/forseti-security/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by mirons-google** [comment]
@carise @blueandgold any other changes here?

**Comment by blueandgold** [comment]
Can you also please add a new row, or Folders IAM policies, both inventory and scanner.

**Comment by carise** [comment]
Folder IAM policies is there already :)

**Comment by carise** [comment]
Same with LB forwarding rules

**Comment by carise** [comment]
Is BigQuery scanner done?

**Comment by mirons-google** [comment]
Done! PTAL.

File: _includes/_inventory/resources.md [hunk 1]
```diff
@@ -7,23 +7,23 @@ interested in isn't listed, please open an issue or contribute!
...
+| [Compute Engine Instance Templates](https://cloud.google.com/compute/docs/reference/latest/instanceTemplates) | Done | Done | TODO |
 | [Firewalls](https://cloud.google.com/compute/docs/reference/latest/firewalls) | Done | TODO | Done |
 | [Folders](https://cloud.google.com/resource-manager/reference/rest/v2beta1/folders) | Done | Done | TODO |
 | [Folders Cloud IAM Policies](https://cloud.google.com/iam/reference/rest/v1/Policy) | Done | Done | TODO |
 | [GSuite Groups](https://developers.google.com/admin-sdk/directory/v1/guides/manage-groups) | Done | Done | TODO |
 | [GSuite Group Members](https://developers.google.com/admin-sdk/directory/v1/guides/manage-group-members) | Done | Done | TODO |
-| [IAM GrantableRoles](https://cloud.google.com/iam/reference/rest/v1/roles/queryGrantableRoles) | In Progress | TODO | TODO |
-| [Load Balancer Forwarding Rules](https://cloud.google.com/compute/docs/reference/latest/forwardingRules#resource) | Done | In Progress | TODO |
+| [IAM GrantableRoles](https://cloud.google.com/iam/reference/rest/v1/roles/queryGrantableRoles) | Done | TODO | TODO |
```
Comment (blueandgold) [inline _includes/_inventory/resources.md]: Sorry for my fault for not being clear, this is inventory and is still in progress.

**Comment by mirons-google** [comment]
done!

**Comment by blueandgold** [comment]
Adding `cla: yes` label, as the PR author is a googler at the time of this PR was created.

**Pull request merged**